calculation/gmhazard_calc/gmhazard_calc/dbs/SiteSourceDB.py

```diff
@@ -51,7 +51,7 @@ def stored_stations(self):
             self._keys_cache = set(self._db.keys())
 
         return [
-            cur_key.split("_")[-1]
```

Comment:
I'd put a comment here to explain what it's doing / the structure of a key. "removes the '/distance/station_' at the start of the key to retrieve the station name"

It's out of your scope but I'll put "/distance" into a constant as it's starting to be referred to multiple times in this code.

Reply:
@claudio525 In case you missed it :smiley:

Reply:
@SnowNooDLe Can you add that comment and add that constant pls